counsel.el

```diff
@@ -1812,11 +1812,25 @@ currently checked out."
     (define-key map [remap undo] 'counsel-find-file-undo)
     map))
 
+;;* File
+;;** `counsel-file-jump'
+(defvar counsel-file-jump-map
+  (let ((map (make-sparse-keymap)))
+    (define-key map (kbd "`") #'counsel-file-find-from-jump)
+    map))
+
 (defun counsel-file-jump-from-find ()
   "Switch to `counsel-file-jump' from `counsel-find-file'."
   (interactive)
   (ivy-quit-and-run
-    (counsel-file-jump ivy-text)))
+    (counsel-file-jump ivy-text (ivy-state-directory ivy-last))))
+
+(defun counsel-file-find-from-jump ()
```

Comment:
Shouldn't this be called `counsel-find-file-from-jump`?

Reply:
I now understand the naming convention. I have fixed that.